src/tests/ftest/control/dmg_system_reformat.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+from __future__ import print_function
+
+import os
+
+from avocado import fail_on
+from apricot import TestWithServers
+from server_utils import ServerFailed
+from command_utils import CommandFailure
+from general_utils import human_to_bytes
+
+
+class DmgSystemReformatTest(TestWithServers):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description:
+
+    Test to verify dmg storage format reformat option works as expected on a
+    DAOS system after a controlled shutdown.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up for DmgSystemReformatTest test."""
+        super(DmgSystemReformatTest, self).setUp()
+
+        host = self.hostlist_servers[0]
+        scm_list = self.server_managers[-1].get_config_value("scm_list")
+        pmem_ns = os.path.split(scm_list[-1])
+        storage_info = self.get_dmg_command().storage_scan(verbose=True)
+        self.scm_cap = storage_info[host]["scm"][pmem_ns]["capacity"]
+
+    @fail_on(CommandFailure)
+    def create_pool_at_capacity(self, percentage):
+        """Create a pool at specified capacity.
+
+        Args:
+            percentage (int): percent to use of total space available for pool.
+                i.e. 0.5 (50%) 0.25(25%)
+
+        Raises:
+            CommandFailure: raise exception if pool creation fails.
+
+        Returns:
+            TestPool: the created test pool object.
+
+        """
+        if percentage > 1 or percentage <= 0:
+            self.fail("The percent value provided cannot be used: %s percent",
+                      percentage)
+        # Convert info from dmg's human readable to bytes
+        scm_size_bytes = human_to_bytes(self.scm_cap)
+        scm_size = percentage * scm_size_bytes
+
+        # Create pool object, update size value with unit and create.
+        self.log.info("Create pool at {:.0%} capacity.".format(percentage))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments